drf_braces/fields/custom.py

```diff
@@ -73,15 +73,18 @@ class RoundedDecimalField(fields.DecimalField):
     to two decimal places.
     """
 
-    def __init__(self, max_digits=100, decimal_places=2, *args, **kwargs):
-        super(fields.DecimalField, self).__init__(
+    def __init__(self, max_digits=None, decimal_places=2, *args, **kwargs):
+        super(RoundedDecimalField, self).__init__(
             max_digits=max_digits,
             decimal_places=decimal_places,
             *args, **kwargs
         )
 
-    def to_internal_value(self, data):
-        return self.quantize(super(RoundedDecimalField, self).to_internal_value(data))
```

Comment:
older versions of DRF dont always call `quantize` on validation https://github.com/encode/django-rest-framework/blob/3.2.5/rest_framework/fields.py#L935